handbook/handbook.in

```diff
@@ -521,7 +521,7 @@ single partition is currently not possible.</p>
 
 <hr />
 
-<p><img src="images/installer-screenshot.png" alt="" /></p>
+<p><img src="handbook/handbook/images/installer-screenshot.png" alt="" /></p>
```

Comment:
Same here.

Reply:
Thanks for the great feedback! Fixed :-)